apps/protocol-frontend/src/utils/arrays.ts

```diff
@@ -0,0 +1,9 @@
+import { UIContribution } from '@govrn/ui-types';
+
+export const mergePages = (pages: UIContribution[][]) => {
```

Comment:
`mergePages` is to specific or coupled to single use-case: `UIContribution[][]`. It could be generic and have general naming.